src/main/kotlin/com/codecraft/ai/api/engine/Constants.kt

```diff
@@ -0,0 +1,16 @@
+package com.codecraft.ai.api.engine
+
+const val GREETINGS = "This Is Key fusion API (Host of Stable Diffusion!) V=1.0.0"
+
+const val StableDiffusionLocalhost = "http://127.0.0.1:7860/sdapi/v1/txt2img"
+
+enum class QueryParam(val strRepresentation: String) {
```

Comment:
strRepresentation does not look like a good name. Maybe something like `value` or `key`

Reply:
Ok let it be value